Review ch2 (#169)
worldbank/d4di: chapters/handling-data.tex

This may mean applying your techniques to their data
or implementing a similar structure in a different context.
As a pure public good, this is nearly costless.

> The useful tools and standards you create will have high value to others 

Reviewer: ```suggestion
The useful tools and standards you create will have high value to others.
```

> (if you are personally or professionally motivated by citations,

Reviewer: ```suggestion
If you are personally or professionally motivated by citations,
```

producing these kinds of resources can lead to that as well).
Therefore, your code should be written neatly and published openly.
It should be easy to read and understand in terms of structure, style, and syntax.
Finally, the corresponding dataset should be openly accessible

> unless for legal or ethical reasons it cannot (we will discuss more about this soon).\sidenote{\url{https://dimewiki.worldbank.org/wiki/Publishing_Data}}

Reviewer: ```suggestion
unless for legal or ethical reasons it cannot.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Publishing_Data}}
```

There are various software solutions for building documentation over time.
The \textbf{Open Science Framework}\sidenote{\url{https://osf.io/}} provides one such solution,
with integrated file storage, version histories, and collaborative wiki pages.
\textbf{GitHub}\sidenote{\url{https://github.com}} provides a transparent documentation system,\sidenote{\url{https://dimewiki.worldbank.org/wiki/Getting_started_with_GitHub}}
\index{task management}\index{GitHub}
in addition to version histories and wiki pages.

> It offers multiple different ways to justify changes and additions, 

Reviewer: ```suggestion
It offers multiple different ways to record the decision process leading to changes and additions, 
```
